components/runners/ambient-runner/ambient_runner/app.py

```
@@ -317,6 +323,38 @@ def _get_workflow_startup_prompt() -> str:
+        from ambient_runner.endpoints.auto_analysis import run_auto_analysis
+
+        repos = get_repos_config()
+        if not repos:
+            return
+
+        for repo in repos:
+            name = repo.get("name", "")
+            url = repo.get("url", "")
+            if name and url:
+                await run_auto_analysis(name, url, bridge)
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)